bazel/repositories.bzl

```diff
@@ -338,10 +338,10 @@ def _com_github_google_libprotobuf_mutator():
         build_file = "@envoy//bazel/external:libprotobuf_mutator.BUILD",
     )
 
-def _com_github_jbeder_yaml_cpp():
-    location = _get_location("com_github_jbeder_yaml_cpp")
+def _com_github_greenhouse_org_yaml_cpp():
+    location = _get_location("com_github_greenhouse_org_yaml_cpp")
     http_archive(
-        name = "com_github_jbeder_yaml_cpp",
+        name = "com_github_greenhouse_org_yaml_cpp",
```

Comment:
Don't rename this, this has to match what in `https://github.com/greenhouse-org/yaml-cpp/blob/master/WORKSPACE#L1`.